docs/operate/monitoring/tracing.mdx

```diff
@@ -11,84 +11,71 @@ Restate supports the following tracing features:
-Start Jaeger with the environment variable `COLLECTOR_OTLP_ENABLED=true` to enable OTLP.
-</Admonition>
-
-Refer to the [Jaeger documentation](https://www.jaegertracing.io/docs/1.46/deployment/) for more details on how to deploy Jaeger.
-
-<Admonition type="note">
-Configure the tracing endpoint in Restate as a fully specified URL: `http://<jaeger-collector>:4317`.
-</Admonition>
-
-You can configure a span/event filter in a similar fashion to the [log filter](/operate/monitoring/restate_logging#log-filter) setting the `observability.tracing.filter` configuration entry.
-
-#### Try it out locally
-To experiment with this feature locally, run the Jaeger container via:
 ```shell
 docker run -d --name jaeger \
+# withClass highlight-line
     -e COLLECTOR_OTLP_ENABLED=true \
-    -p 4317:4317 \
-    -p 16686:16686 \
+    -p 4317:4317 -p 16686:16686 \
     jaegertracing/all-in-one:1.46
 ```
 
-Then launch Restate with the tracing endpoint defined as an environment variable:
+Configure the tracing endpoint in Restate as a fully specified URL.
+For example, locally with Docker:
 
 ```shell
-docker run --name restate_dev --rm \
+docker run -p 8080:8080 -p 9070:9070 -p 9071:9071 \
+# withClass highlight-line
   -e RESTATE_OBSERVABILITY__TRACING__ENDPOINT=http://host.docker.internal:4317 \
-  -p 8080:8080 -p 9070:9070 -p 9071:9071 \
   --add-host=host.docker.internal:host-gateway \
   docker.io/restatedev/restate:VAR::RESTATE_VERSION
 ```
 
-Go to the Jaeger UI at http://localhost:16686.
-
-If you now spin up your Restate services and send requests to them, you will see the traces appear in the Jaeger UI.
+If you now spin up your services and send requests to them, you will see the traces appear in the Jaeger UI at http://localhost:16686:
 
-An example from the [shopping cart example](https://github.com/restatedev/examples/tree/main/end-to-end-applications/typescript/ecommerce-store):
-![Observability](/img/observability.jpeg)
+<img src="/img/observability.jpeg" width={"60%"}/>
 
-## Setting up Jaeger file exporter
+<Admonition type="info" title={"Filtering traces"}>
+    You can configure a span/event filter in a similar fashion to the [log filter](/operate/monitoring/restate_logging#log-filter) setting the `observability.tracing.filter` configuration entry.
+</Admonition>
 
-If you can't configure a Jaeger agent, you can export traces by writing them to files, using the Jaeger JSON format. To do so, set up the configuration entry `observability.tracing.json_file_export_path` pointing towards the path where trace files should be written.
+<details className={"grey-details"}>
+    <summary>Setting up Jaeger file exporter</summary>
 
-You can import the trace files using the Jaeger UI:
+    If you can't configure a Jaeger agent, you can export traces by writing them to files, using the Jaeger JSON format. To do so, set up the configuration entry `observability.tracing.json_file_export_path` pointing towards the path where trace files should be written.
```

Comment:
`observability.tracing.json_file_export_path` should no longer be correct. Now it should be `tracing-json-path`.